### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by jacoblololol** [alekxeyuk/Skribbl.io-Bot#15]
Am I able to see what it is drawing?

**Comment by GrimReaperFloof** [comment]
Nope. It could be implementable somehow though. But when it draws it already gives you a link of whatever image it's going to draw
